### PR TITLE
Deprecate watch in favor of watch_port.

### PR DESCRIPTION
**Pull request by stefanheule**
Fixes https://github.com/p4lang/p4runtime/issues/307

### CODE REVIEW

File: docs/v1/P4Runtime-Spec.mdk [hunk 1]
```diff
@@ -6071,6 +6072,11 @@ man-in-the-middle attacks between the server and client.
 
 ## Revision History
 
+### Changes in v1.3.0
+
+* Deprecate `watch` field (uint32) in favor of `watch_port` (bytes). This allows
```
Comment (antoninbas) [inline docs/v1/P4Runtime-Spec.mdk:6077]: nit: the previous type was int32, not uint32

**Review by antoninbas — COMMENTED** [review]
one nit, otherwise LGTM

File: docs/v1/P4Runtime-Spec.mdk (hunk 1 above)
Comment (stefanheule) [reply docs/v1/P4Runtime-Spec.mdk]: Fixed.

**Review by antoninbas — COMMENTED** [review]
LGTM. Could you resolve the merge conflict and address the CI error?

**Comment by stefanheule** [comment]
> LGTM. Could you resolve the merge conflict and address the CI error?

It appears the build fails with a go problem? Any idea what is wrong? I didn't touch that at all. https://travis-ci.com/github/p4lang/p4runtime/builds/189092852

**Comment by antoninbas** [comment]
From the CI logs:

> The generated Go files are not up-to-date
> You can regenerate them with './codegen/update_go.sh' and commit the changes

**Comment by stefanheule** [comment]
> From the CI logs:
> 
> > The generated Go files are not up-to-date
> > You can regenerate them with './codegen/update_go.sh' and commit the changes

Oops, sorry, the fact that this was talking about go and I didn't change any go threw me off. But now I understand. Fixed.

**Comment by antoninbas** [comment]
The diff is showing changes from earlier PRs, could you rebase? I want to make sure I merge the right thing

**Comment by stefanheule** [comment]
> The diff is showing changes from earlier PRs, could you rebase? I want to make sure I merge the right thing

Done.

**Review by antoninbas — APPROVED** [review]
LGTM

**Pull request merged**